benchmark/benchmark_block_run_length_decode.cpp

```diff
@@ -171,26 +171,37 @@ void run_benchmark(benchmark::State& state, hipStream_t stream, size_t N)
     HIP_CHECK(hipFree(d_output));
 }
 
-#define CREATE_BENCHMARK(IT, OT, MINRL, MAXRL, BS, RPT, DIPT)                                 \
-    benchmark::RegisterBenchmark("block_run_length_decode<Item Type:" #IT ",Offset Type:" #OT \
-                                 ",Min RunLength:" #MINRL ",Max RunLength:" #MAXRL            \
-                                 ",BlockSize: " #BS ",Runs Per Thread:" #RPT                  \
-                                 ",Decoded Items Per Thread:" #DIPT ">",                      \
-                                 &run_benchmark<IT, OT, MINRL, MAXRL, BS, RPT, DIPT>,         \
-                                 stream,                                                      \
-                                 size)
+#define CREATE_BENCHMARK(IT, OT, MINRL, MAXRL, BS, RPT, DIPT)                                   \
+    benchmark::RegisterBenchmark(bench_naming::format_name("{lvl:block,algo:run_length_decode"  \
+                                    ",item_type:" #IT                                           \
+                                    ",offset_type:" #OT                                         \
```

Comment:
I think it might be nicer to also standarize field names with https://github.com/ROCm/hipCUB/pull/368. With this change it is lowercase snakecake in rocPRIM and capatalized with spaces in hipCUB. Personally I think snake case might be better to use as handling spaces is always a pain

Reply:
Okay sounds good, I will make it snakecase in hipCUB as well